FirebaseAuth/Tests/Unit/RPCBaseTests.swift

```diff
@@ -68,15 +68,16 @@ class RPCBaseTests: XCTestCase {
   let kTestIdentifier = "Identifier"
 
   var rpcIssuer: FakeBackendRPCIssuer!
+  var authBackend: AuthBackend!
 
   override func setUp() {
     rpcIssuer = FakeBackendRPCIssuer()
-    AuthBackend.setTestRPCIssuer(issuer: rpcIssuer)
+    authBackend = AuthBackend(rpcIssuer: rpcIssuer)
```

Comment:
I think this approach is a lot cleaner for testability! Nice to see `setTestRPCIssuer` get deleted above.

Reply:
Thanks for the review!